### DISCUSSION TITLE
API python library to work with https

### DISCUSSION THREAD

**Discussion by jesseyu1984918** [inventree/InvenTree#8101] [Q&A]
Hi,
   does the API python library works with HTTPS? I changed the server address to HTTPS and it doesn't seem to work anymore, is there any settings needs to be done?

**Comment by SchrodingersGat** [comment]
> I changed the server address to HTTPS and it doesn't seem to work anymore

Please provide some more information on what "not work anymore" means. Any error messages?

**Comment by jesseyu1984918** [comment]
If I use a HTTPS path, it can’t be authenticatedOn Sep 9, 2024, at 5:16 PM, Oliver ***@***.***> wrote:﻿

I changed the server address to HTTPS and it doesn't seem to work anymore

Please provide some more information on what "not work anymore" means. Any error messages?

**Reply by SchrodingersGat** [reply]
Are you using a self-signed https certificate, or one that might fail strict checks?

**Comment by jesseyu1984918** [comment]
I’m not expert on this but I think it’s a self-signed https certificateOn Sep 9, 2024, at 5:26 PM, Oliver ***@***.***> wrote:﻿
Are you using a self-signed https certificate, or one that might fail strict checks?

**Comment by SchrodingersGat** [comment]
@jesseyu1984918 I have just released [version 0.17.0](https://pypi.org/project/inventree/) of the python bindings, when you are creating the `InvenTreeAPI` object you can now pass the `strict=False` parameter. Please give this a go and see if it makes a difference?
